python_agent/test_custom_env.py

```
@@ -0,0 +1,154 @@
+
+# Important: Add python_agent to sys.path if tests are run from root or another directory
+# For simplicity, assume tests might be run from within python_agent directory or PYTHONPATH is set.
+# If running with `python -m unittest discover ./python_agent` from root, imports should work.
+try:
+    from custom_env import CustomGameEnv, MAX_NEARBY_NPCS, MAX_INVENTORY_ITEMS, MAX_GROUND_ITEMS, BONE_ITEM_ID
+except ImportError:
+    # Fallback for running directly from python_agent or if path issues occur
+    import sys
+    import os
+    sys.path.append(os.path.dirname(os.path.abspath(__file__)))
+    from custom_env import CustomGameEnv, MAX_NEARBY_NPCS, MAX_INVENTORY_ITEMS, MAX_GROUND_ITEMS, BONE_ITEM_ID
```

Ruff (0.11.9)

`custom_env.MAX_NEARBY_NPCS` imported but unused

Remove unused import

(F401)